rust/src/tx_builder.rs

```diff
@@ -4247,5 +4461,188 @@ mod tests {
         assert_eq!(ma2_output.get(&policy_id2).unwrap().get(&name).unwrap(), to_bignum(40));
     }
 
+    fn create_base_address_from_script_hash(sh: &ScriptHash) -> Address {
```

Comment:
I know name is long, but it's better add 'test' or 'testnet' to name for separate from main logic